tests/fork-change-slot.c

```diff
@@ -0,0 +1,347 @@
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <fcntl.h>
+#include <termios.h>
+#include <stdio.h>
+#include <string.h>
+#include <sys/types.h>
+#include <sys/wait.h>
+#include <unistd.h>
+#include <signal.h>
+
+#include <execinfo.h>
+
+#include <openssl/evp.h>
+#include <openssl/x509.h>
+#include <openssl/bio.h>
+#include <openssl/pem.h>
+#include <openssl/engine.h>
+
```

Comment:
A small summary of what this program does would be nice. Think of it of a descritpion for someone trying to figure what this program does few years later.

Reply:
Ok. I'll add a description for the program